nmkapp/forms.py

```diff
@@ -81,6 +81,46 @@ def clean(self):
         return cleaned_data
 
 
+class SetPasswordForm(forms.Form):
+    """
+    A form that lets a user change set their password without entering the old
+    password
+    """
+    error_messages = {
+        'password_mismatch': _("The two password fields didn't match."),
+    }
+    new_password1 = forms.CharField(label=_("New password"),
+                                    max_length=28,
+                                    min_length=5,
+                                    widget=forms.PasswordInput)
+    new_password2 = forms.CharField(label=_("New password confirmation"),
+                                    max_length=28,
+                                    min_length=5,
+                                    widget=forms.PasswordInput)
+
+    def __init__(self, user, *args, **kwargs):
+        self.user = user
+        super(SetPasswordForm, self).__init__(*args, **kwargs)
+
+    def clean_new_password2(self):
+        password1 = self.cleaned_data.get('new_password1')
+        password2 = self.cleaned_data.get('new_password2')
+        if password1 and password2:
```

Comment:
sto ovaj if? sta ako je password1 "foo", a password2 None - jel onda to treba da prodje kao ispravno?:)

Reply:
To je djangova form-a :/
Kopirao sam je jer mi je trebalo da je editujem malo, ali taj deo nisam dirao

Reply:
https://github.com/django/django/blob/stable/2.0.x/django/contrib/auth/forms.py

Reply:
oket, onda bar dopisi u komentar da je ukradena:)